assets/dev/js/editor/container/container.js

```diff
@@ -238,6 +238,15 @@ export default class Container extends ArgsObject {
 		}
 	}
 
+	/**
+	 * Function addRepeaterItem().
+	 *
+	 * @param {string} repeaterName
+	 * @param {Backbone.Model} rowSettingsModel
+	 * @param {number} index
+	 *
+	 * @returns {Container}
+	 */
```

Comment:
Nice JSDoc :)
Maybe add a function description, and maybe even parameters description if needed.